test/integration/style-spec/validate_spec.test.ts

```diff
@@ -19,10 +19,80 @@ describe('validate_spec', () => {
     });
 
     test('errors from validate do not contain line numbers', () => {
-        const style = JSON.parse(fs.readFileSync('test/integration/style-spec/tests/bad-color.input.json', 'utf8'));
+        const style = JSON.parse(
+            fs.readFileSync(
+                'test/integration/style-spec/tests/bad-color.input.json',
+                'utf8'
+            )
+        );
 
         const result = validate(style, reference);
         expect(result[0].line).toBeUndefined();
     });
 
+    test('Validate sdk-support in spec', () => {
```

Comment:
You can create the tests dynamically like done above so you can have the test name indicate which definition in the spec failed. This is just a though.

Reply:
Done